includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -177,7 +177,7 @@ public function get_settings(): WP_REST_Response {
 				'available_payment_method_ids'      => $this->wcpay_gateway->get_upe_available_payment_methods(),
 				'is_wcpay_enabled'                  => $this->wcpay_gateway->is_enabled(),
 				'is_manual_capture_enabled'         => 'yes' === $this->wcpay_gateway->get_option( 'manual_capture' ),
-				'is_test_mode_enabled'              => 'yes' === $this->wcpay_gateway->is_in_test_mode(),
+				'is_test_mode_enabled'              => $this->wcpay_gateway->is_in_test_mode(),
```

Comment:
The whole issue is that the `is_in_test_mode` method does not return a string, but a boolean